setup.py

```diff
@@ -66,14 +66,19 @@
 setup(
     name='wagtail',
     version=__version__,
-    description='A Django content management system focused on flexibility and user experience',
+    description='A Django content management system.',
     author='Matthew Westcott',
     author_email='matthew.westcott@torchbox.com',
     url='http://wagtail.io/',
     packages=find_packages(),
     include_package_data=True,
     license='BSD',
-    long_description=open('README.rst').read(),
+    long_description="Wagtail is an open source content management \
```

Comment:
# Markdown Long Description Done Right

```python
long_description=open('README.md').read(),
long_description_content_type='text/markdown',  # This is important!
```

Done!

## Note

  If, in the previous step(`sdist/bdist_wheel`), you also created a wheel, perhaps with python setup.py bdist_wheel, you will need to upload your distributions one at a time, with the source distribution first, to avoid an issue in wheel which should be fixed soon. E.g.:
```
$ twine upload dist/<filename>.tar.gz
$ twine upload dist/<filename>.whl
```
Otherwise, this is also the same as usual:

`$ twine upload dist/*`

FYI: https://github.com/di/markdown-description-example/blob/master/setup.py
https://dustingram.com/articles/2018/03/16/markdown-descriptions-on-pypi

Reply:
Wahooooo! This is amazing, thank you for taking the time to let us know here @drunkwcodes 🎉🎉

Reply:
I found an occasion to play with this Markdown support over at https://github.com/springload/draftjs_exporter/pull/94, and can officially report back that it's super cool 👏👏👏